stats/cloud/client.go

```diff
@@ -201,3 +223,12 @@ func shouldRetry(resp *http.Response, err error, attempt, maxAttempts int) bool
 
 	return false
 }
+
+func generateSessionToken() (string, error) {
+	sessionToken, err := uuid.NewV4()
+	if err != nil {
+		return "", err
+	}
+
+	return sessionToken.String(), nil
```

Comment:
I really don't like that now we have an error that we have to check that 
1. we have no way of fixing
2. it's very unlikely that `crypto/rand#Read` will return an error >.> 

But unfortunately I don't have any proposals ...

Reply:
I often use https://github.com/satori/go.uuid in all my previous projects. Since when I see current uuid package inside vendor already, I don't want to touch it in this PR.

We can ignore it for now and I will create separate issue for changing uuid package.

How do you think @na-- @imiric ?

Reply:
I don't think changing to a different uuid package will help us with the error ... the problem is `crypto.rand#Read` can return an error and all of those use them AFAIK.
My argument is that we don't need `crypto/rand` at all and the non crypto one always returns a `nil` error :). But I don't think it's that big of problem ... I just really dislike that we added another error that is 
1. unlikely
2. we have 0 things we can do, but report it or retry ... maybe
3. can't test

Reply:
@MStoykov not sure what's your point. My point ios that satori/go.uuid *does* not return error when creating new uuid string, so we don't have to check for error.

Reply:
Hmm, maybe I should change the current uuid package, as I see it is not used anywhere in current code base (not sure why it existed in vendor)

Reply:
It's used by a dependancy of a dependancy of a dependancy ... connected to #911 
as far as I can see the ones don't return an error take their random data from us ... at which point I would argue that we should just use the non-crypro rand, generate a value and hex encode it and call it day. 

AFAIK we don't really need a UUIDvX we need random enough string that won't be repeated for the same user if they run another test ... soon after the previous one

Reply:
> as far as I can see the ones don't return an error take their random data from us

Can you explain more details?

I'm ok with not adding another dependency, and roll our own one (but at least, using crypto/rand instead of math/rand, and panic if error return). With legal user, non-crypto rand is ok for us. But illegal one which can force k6 to behave wrongly if we don't use a strong way to generate the token.

Your idea @na-- @imiric ?

Reply:
Okay ... so apparently that lib you just added has a [last version](https://github.com/satori/go.uuid/commit/f58768cc1a7a7e77a3bd49e98cdd21419399b6a3) which panics on error and the very [next commit](https://github.com/satori/go.uuid/commit/0ef6afb2f6cdd6cdaeee3885a95099c63f18fc8c) (nearly 2 years ago) adds an error to be returned, which is what is shown in the [godoc](https://godoc.org/github.com/satori/go.uuid#NewV4). 

I am very strongly against adding another UUID lib ... just for this. 

I am also very opposed to panicing. I have no idea when `crypto/rand` could return an error but still. 

For me using `crypto/rand` for this particular token is an overkill .. if it wasn't for the fact we add an error I would've not cared :) , but that is what is bugging me - the error(or panic with this new lib), not the fact we use `crypto` for it

Reply:
@MStoykov note that even you use `math/rand`, its `rand.Read` still returns an error, too.

Reply:
@MStoykov Here's what I often do:

```
func generateSessionToken() string {
	// 16 hex characters
	b := make([]byte, 8)
	if _, err := rand.Read(b); err != nil {
		panic(err)
	}
	return hex.EncodeToString(b)
}
```

Reply:
`https://golang.org/pkg/math/rand/#Rand.Read` it always return a `nil` error, so we can ignore it safely knowing it's never going to happen.

Reply:
@MStoykov good point 👍